indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamIndexTaskRunner.java

```diff
@@ -375,7 +380,7 @@ private TaskStatus runInternal(TaskToolbox toolbox) throws Exception
         // Sanity checks.
         if (!restoredNextPartitions.getStream().equals(ioConfig.getStartPartitions().getStream())) {
           throw new ISE(
-              "WTF?! Restored stream[%s] but expected stream[%s]",
+              "WTF?! Restored topic[%s] but expected topic[%s]",
```

Comment:
OK, I'll revert these changes, but I do think it's better w/ Kafkaesque terminology (https://github.com/apache/incubator-druid/pull/7267#issuecomment-473134915). Especially because "sequence" already means something else in the context of seekable stream tasks (SequenceMetadata, sequenceName, etc) and so it is best to avoid. But this can be driven separately and doesn't need to be looped into this logic adjustment PR.